rosidl_parser/rosidl_parser/__init__.py

```diff
@@ -445,8 +445,7 @@ def parse_value_string(type_, value_string):
     if type_.is_primitive_type() and not type_.is_array:
         return parse_primitive_value_string(type_, value_string)
 
-    # TODO(mikaelarguedas) change this condition once string escape function is implemented
-    if type_.is_primitive_type() and type_.is_array and type_.type != 'string':
+    if type_.is_primitive_type() and type_.is_array:
```

Comment:
It looks like this just enables default values for string arrays but without implementing the required string escaping logic? Currently I simply splits on commas - I don't think this is sufficient since the comma can be within a string.